### DISCUSSION TITLE
INVALID_DEDUCTION_AMOUNT - Arányosítással számolt visszaigényelhető adót tartalmazó tétel sztornózása

### DISCUSSION THREAD

**Discussion by rmskga** [nav-gov-hu/eVAT#122] [Kérdések és válaszok - Gépi interfész]
Levonható tétel esetén a következő részletet tartalmazó XML-t elfogadja a feldolgozó.

```
<n1:taxInformation>
    <n1:standardTaxCode>MD66</n1:standardTaxCode>
    <n1:ownTaxCode>66</n1:ownTaxCode>
    <n1:taxPosition>
        <n1:positionType>DEDUCTIBLE</n1:positionType>
        <n1:taxBase>200</n1:taxBase>
        <n1:taxAmount>54</n1:taxAmount>
        <n1:deductionOptions>
            <n1:deductionAmount>27</n1:deductionAmount>
        </n1:deductionOptions>
    </n1:taxPosition>
</n1:taxInformation
```

Sztornóként, negatív előjelet használva viszont INVALID_DEDUCTION_AMOUNT hibaüzenetet kapunk, mivel a taxAmount < deductionAmount.

```
<n1:taxInformation>
    <n1:standardTaxCode>MD66</n1:standardTaxCode>
    <n1:ownTaxCode>66</n1:ownTaxCode>
    <n1:taxPosition>
        <n1:positionType>DEDUCTIBLE</n1:positionType>
        <n1:taxBase>-200</n1:taxBase>
        <n1:taxAmount>-54</n1:taxAmount>
        <n1:deductionOptions>
            <n1:deductionAmount>-27</n1:deductionAmount>
        </n1:deductionOptions>
    </n1:taxPosition>
</n1:taxInformation
```
Ebben az esetben mi lenne a sztornózás helyes módja?

**Comment by NTCA-KJ** [comment]
Tisztelt Ügyfelünk!

Levonható adót csökkentő, negatív előjelű tétel esetén az MP31 standard adókód alkalmazandó.
Az áfabevallás 31. sorának (a) oszlopában kizárólag pozitív előjelű érték szerepeltethető, ezért az XML állományban az MP31 adókódhoz tartozó adóalap- és adóösszeg-adatokat szintén pozitív értékként szükséges megadni.

<n0:taxInformation>
				<n0:standardTaxCode>MP31</n0:standardTaxCode>
				<n0:ownTaxCode>V5</n0:ownTaxCode>
				<n0:taxPosition>
					<n0:positionType>PAYABLE</n0:positionType>
					<n0:taxBase>100000</n0:taxBase>
					<n0:taxAmount>27000</n0:taxAmount>
				</n0:taxPosition>
</n0:taxInformation>

A pozitív előjelű értékek alkalmazására vonatkozó szabály az eÁFA M2M interfész specifikáció következő verziójában kerül részletesen dokumentálásra.
Ezzel egyidejűleg az MP31 adókódhoz kapcsolódó, jelenleg fennálló validációs ellenőrzések módosítására is sor kerül annak érdekében, hogy azok összhangban legyenek a fenti adattartalmi követelménnyel. Így a javítást követően "INVALID_DEDUCTION_AMOUNT" hiba is törlésre kerül ezen adókódnál.

**Reply by TomaszSzelaTR** [reply]
Jól értem, hogy:
Az értékesítéshez kapcsolódó jóváíró számlák (kimenő ÁFA) negatív összeggel jelenthetők, ahol a positionType = PAYABLE.
A beszerzéshez kapcsolódó jóváíró számlákat (bejövő ÁFA) mindig pozitív összeggel kell jelenteni, a standardTaxCode = MP31 és a positionType = PAYABLE használatával.
Helyes ez az értelmezés?

---------------
Am I correct in understanding that:
Credit notes related to sales (output VAT) should be reported as negative amounts with positionType = PAYABLE.
Credit notes related to purchases (input VAT) should always be reported as positive amounts using standardTaxCode = MP31 and positionType = PAYABLE.
Is that interpretation justified?

**Reply by Szildiko** [reply]
Bejövő számlához kiegészítés, akkor ha az alapszámlához képest a jóváíró számlát eltérő bevallási időszakban szerepelteted.
Ha az alapszámla és a jóváíró számla ugyanabban a bevallásban kerül jelentésre, akkor nem kell MP31, hanem marad az MD6X.

**Comment by Macskafarka** [comment]
Nem lehetne elfelejteni az MP31-et?
Ügyviteli program oldaláról kezelhetetlen dolog az, hogy változhat az adókód MD66-ról MP31-re akkor is, ha a bizonylaton nem történik semmi módosítás, csupán a bevallási gyakoriság változása miatt a lánckezdő számla másik (korábbi) bevallási időszakba esik, mint a levonható áfát utólag negatív irányba módosító okirat. 
Adatrögzítéskor nincs sem a felhasználó, sem az ERP rendszer annak a tudásnak a birtokában, hogy MD66 vagy MP31 a "helyes" kód.
Kinek a felelőssége annak felismerése, hogy a már adókóddal ellátott tétel átkódolásra kellene kerülnie?
Az az ügyfeleknek is nagy teher, és a ERP supportot is lefoglalja ennek a magyarázgatása. 
Mindezt annak fényében nehéz megérteni, hogy más adókódoknál lehet negatív érték.

**Comment by Macskafarka** [comment]
Ez az MP31 azért is totális tévút, mert a ha valaki kap egy 1 Mrd Ft összegű számlát, amit bevall, majd annak a helyesbítését 100.000 Ft-ra, amit át kell tenni fizetendőbe MP31 kóddal, akkor kisebb adóalanynál ez így azonnal bevallási gyakoriságváltást ererdményez értékhatár túllépés miatt.
Holott ez az MP31 csupán egy mesterséges kreálmány. A könyvelésében senki nem szerepelteti a negatív levonandót fizetendőként, ugyanúgy 4661-re könyveli és nem 467-re

**Comment by Szildiko** [comment]
"Ügyviteli program oldaláról kezelhetetlen dolog"
Nem kezelhetetlen, mi is kezeljük, az ügyfélnek megfelelő áfakódot kell választania.
Pl nálunk van egy 570-es kód, ami a 31-es sorra dolgozik. Az ügyfélnek kell tisztában lennie a törvény szabályával, hogy mikor kell az áfa bevallás 31. sorába adatot raknia és akkor ezt a kódot kell alkalmaznia, ilyenkor nem mínusz értékkel kell beírni az adatot, hanem pozitívval.
és ilyenkor a 467-t kell használnia.

Másik programunk is kezeli automatizmussal, ott ha hivatkozás van és látja hogy az alap számla áfa dátuma eltér a helyesbítő áfa dátumától, akkor tudja hogy át kell tennie a 31-es sorba. (itt vannak egyéb problémáink, de az nem a NAV gondja, hanem a miénk, hogy miért úgy alakítottuk ki ahogy)

**Comment by Macskafarka** [comment]
Kedves @Szildiko! 
Ezt írtad:  "látja hogy az alap számla áfa dátuma eltér a helyesbítő áfa dátumától, akkor tudja hogy át kell tennie a 31-es sorba."
Önmagában az áfa dátum eltérése nem okozza a 31. sorba kerülést, és épp ez az én problémám.
Ha a számla áfa dátuma (taxpointDate) 2026.08.11. a helyesbítőé 2026.08.12, akkor általában nem kell áttenni a 31. sorba, mert általában ugyanabba az áfa bevallásba kerülnek.  
Ha viszont a számla áfa dátuma 2026.07.25, és a helyesbítőé 2026.08.12. akkor ha negyedéves áfa bevalló, akkor szintén nem kell áttenni a 31. sorba, mert ugyanabba a bevallásba kerülnek.
Ha azonban bevallás gyakoriság váltás történik, és kiderül, hogy az adózó 2026.07.01-től havi bevallásra kötelezett, és úgy dönt, hogy a számlát beteszi a 07. havi bevallásba, és a helyesbítőt a 08. haviba, akkor azt máris MP31-re kell kódolni. 
A legelső példában, amikor ugyanabba a hónapba esik a két bizonylat áfa dátuma, 1 nap eltéréssel, akkor is kerülhet külön áfa időszakba (pl. ha előtársaságnak soron kívüli bevallást kell benyújtania 2026.08.11-i dátummal,
A problémám az, hogy mi a bizonylat érvényes berögzítéskor adókódolunk, de ezt követően a bizonylat változatlansága mellett is változhat az adókód, aminek a kezelése **elvárható, épeszű ráfordítással** megoldhatatlan.

**Reply by Szildiko** [reply]
Ti esetekben lehet nem, a miénkben igen, mert az alapján gyűjti a programunk az áfa bevallást.
természetesen a programban azt is be kell állítani, hogy milyen bevalló az ügyfél.
ha gyakoriság váltás történik, akkor a cégadatban lekezeli, hogy mikortól más a gyakorisága, és újra generálja az automata bizonylatokat a program

**Reply by Macskafarka** [reply]
Önmagában az a furcsa, és a számomra érthetetlen , hogy a bizonylat adataitól függetlenül is változhat az adókód.

**Reply by Macskafarka** [reply]
Hajlok arra, hogy bekódoljuk simán MD66-ra a negatív összeget, majd a NAV ad rá ERROR-t, és ekkor majd a ügyfél módosítja MP31-re.
Csak halkan jegyzem meg, hogy ezt akár a NAV is megtehetné....

**Reply by Macskafarka** [reply]
A másik ötletem az, hogy a negatív MD64, MD65, MD66 adókat ha kell, ha nem, egyszerűen gondolkodás nélkül áttesszük MP31-re.
Nem tudom, hogy a NAV ezt hibának értékeli-e, de szerintem nem ad rá WARNING-ot.

**Comment by tilia74** [comment]
Tisztelt Adóhatóság!


**2026-aug-12-én adott alábbi válaszuk  nem tér ki a deduction option kezelésére.**

Az én értelmezésem az alábbi.
Ha az eredeti számlában volt deduction option, akkor a stornó számlában nem kell feltüntetni, mivel positionType = "PAYABLE" lehet csak válaszuk szerint az MP31-es standardTaxCode használata mellett és "A deductionOptions csomópont kizárólag DEDUCTIBLE áfapozíció esetén tölthető ki, és akkor releváns, ha az adóalany az adott tételsorhoz kapcsolódó előzetesen felszámított áfát nem jogosult teljes egészében levonni. Validáció biztosítja, hogy ez a csomópont ne kerülhessen PAYABLE vagy OTHER pozícióhoz rendelt tételsorokra" (forrás: NAV eÁFA M2M rendszer
eÁFA M2M REST API interfészleírás és fejlesztői dokumentáció, 6.3.3.2.4.1. deductionOptions - Levonási adatok, utolsó módosítási dátum: 2026.05.05.)

Ebben az esetben viszont **van két lehetőségünk az XML-ben szerepeltetni a stornó számlát a**z alábbiak szerint.
(1)  A taxBase és taxAmount kalkulációját  módosítani, úgy hogy a taxAmount a helyes, a visszaigényelhető adó értékét csökkentő értéket tartalmazza és ne kapjunk a vatRate-re valamilyen nem létező %-os kulcsot. taxBase = 7000, taxAmount = 1890 vatRate = 0,27
(lásd lent (A) opció) 
(2) A taxBase = 10000 (az eredeti) és taxAmount = 1890 (a visszaigényelhető rész stornó összege) de a vatRate adja 1890/10000 = 0,189 nem megfelelő.
(lásd lent (B) opció) 

Az **(A) és a (B) verzió is felment az eVAT teszt rendszerébe error és warning üzenetek nélkül** 2026.09.17-én.

**Kérdéseim:** 
Melyik változat a helyes a NAV szerint?

Milyen haszna van az MP31-es kód ilyen bonyolult használatának az adóhatóság szempontjából (csak megerősíteni tudom Macskafarka által felvetetteket is), mivel adófizetőként nem látjuk a gazdasági hasznát, mind a követelmények tisztázására, mind az azt követő fejlesztésekre ráfordított erőforrásokkal összevetve?

A NAV tudja, hogy korrekciós, stornó számlát jelent az adófizető az online számla jelentések alapján. Ennek jelölése könnyen meg is oldható az XML-ben az standardTaxCode és positionType váltás nélkül is, ha ez mindenképpen szükséges.
Nagyban támogatnánk a külön standardTaxCode, az MP31-es használatának megszüntetését.


ORIGINAL INVOICE (ennek a teljes jóváírása szerepel a további példákban)

<data:taxInformation>
<data:standardTaxCode>MD66</data:standardTaxCode>
<data:ownTaxCode>52</data:ownTaxCode>
<data:taxPosition>
<data:positionType>DEDUCTIBLE</data:positionType>
<data:taxBase>10000</data:taxBase>
<data:taxAmount>2700</data:taxAmount>
<data:deductionOptions>
<data:deductionRate>0.7</data:deductionRate>
<data:deductionAmount>1890</data:deductionAmount>
</data:deductionOptions>
</data:taxPosition>
</data:taxInformation>

(A) opció XML: 1890/0,27 = 7000 -> ez lesz a taxBase
error üzenet: nincs 

<data:taxInformation>
<data:standardTaxCode>MP31</data:standardTaxCode>
<data:ownTaxCode>CN52</data:ownTaxCode>
<data:taxPosition>
<data:positionType>PAYABLE</data:positionType>
<data:taxBase>7000</data:taxBase>
<data:taxAmount>1890</data:taxAmount>
</data:taxPosition>
</data:taxInformation>

(B) opció XML: taxAmount= 1890 taxBase = 10000 taxRate = 18,9%
error üzenet: nincs 

<data:taxInformation>
<data:standardTaxCode>MP31</data:standardTaxCode>
<data:ownTaxCode>CN52</data:ownTaxCode>
<data:taxPosition>
<data:positionType>PAYABLE</data:positionType>
<data:taxBase>10000</data:taxBase>
<data:taxAmount>1890</data:taxAmount>
</data:taxPosition>
</data:taxInformation>

error üzenet: INVALID_DEDUCTION_AMOUNT - rmskga által felvetett megoldási változat

<data:taxInformation>
<data:standardTaxCode>MD66</data:standardTaxCode>
<data:ownTaxCode>52</data:ownTaxCode>
<data:taxPosition>
<data:positionType>DEDUCTIBLE</data:positionType>
<data:taxBase>-10000</data:taxBase>
<data:taxAmount>-2700</data:taxAmount>
<data:deductionOptions>
<data:deductionRate>0.7</data:deductionRate>
<data:deductionAmount>-1890</data:deductionAmount>
</data:deductionOptions>
</data:taxPosition>
</data:taxInformation>

error üzenet: INVALID_DEDUCTION_OPTIONS - positionType = PAYABLE de van deductionOptions

<data:taxInformation>
<data:standardTaxCode>MP31</data:standardTaxCode>
<data:ownTaxCode>52CN</data:ownTaxCode>
<data:taxPosition>
<data:positionType>PAYABLE</data:positionType>
<data:taxBase>10000</data:taxBase>
<data:taxAmount>2700</data:taxAmount>
<data:deductionOptions>
<data:deductionRate>0.7</data:deductionRate>
<data:deductionAmount>1890</data:deductionAmount>
</data:deductionOptions>
</data:taxPosition>
</data:taxInformation>
